tron/utils/timeutils.py

```diff
@@ -20,6 +20,8 @@ def current_timestamp():
 
 def to_timestamp(time_val):
     """Generate a unix timestamp for the given datetime instance"""
+    if time_val.tzinfo is not None:
+        return time_val.tzinfo.utcoffset(time_val).total_seconds()
```

Comment:
I don't think this is correct. This returns literally "the number of seconds difference between the tzs", not the absolute unit timestamp.

Maybe it is

    time.mktime(time_val.utctimetuple())

?

Reply:
Yes, this works! Thanks. Fixed and added another test that catches this.